_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of me Yaktocat](https://octodex.github.com/images/yaktocat.png)

1. One
1. Two
1. Three